.travis.yml

```diff
@@ -1,5 +1,4 @@
 rvm:
-  - 1.9.3
```

Comment:
Yeah I guess that's fair. They already know they're running on thin ice by using that Ruby and/or Rails.